portfolio/src/main/webapp/leave-comment/script.js

```diff
@@ -0,0 +1,22 @@
+function getComments() {
+  fetch("../comments").
+  then(response => response.text()).
+  then(JSON.parse).
+  then(addCommentsToDom);
```

Comment:
super-nit: the fluent interface pattern looks slightly better when the chained methods are indented

Reply:
Done.